### ISSUE TITLE
MEK dephosphorylates ERK in Biopax

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#42]
Query:

```
from indra.biopax import biopax_api as ba
bp = ba.process_pc_neighborhood(['MAP2K1', 'MAP2K2', 'MAPK1', 'MAPK3'])
bp.get_dephosphorylation()
```

This yields, among others, the statements:

```
Dephosphorylation(MAP2K1(), MAPK3(), PhosphorylationTyrosine, 203),
 Dephosphorylation(MAP2K1(), MAPK3(), PhosphorylationThreonine, 201),
 Dephosphorylation(MAP2K2(), MAPK3(), PhosphorylationTyrosine, 203),
 Dephosphorylation(MAP2K2(), MAPK3(), PhosphorylationThreonine, 201),
 Dephosphorylation(MAP2K1(), MAPK1(), PhosphorylationTyrosine, 184),
 Dephosphorylation(MAP2K1(), MAPK1(), PhosphorylationThreonine, 182),
 Dephosphorylation(MAP2K2(), MAPK1(), PhosphorylationTyrosine, 184),
 Dephosphorylation(MAP2K2(), MAPK1(), PhosphorylationThreonine, 182),
 Dephosphorylation(MAP2K1(), MAPK3(), PhosphorylationTyrosine, 203),
 Dephosphorylation(MAP2K1(), MAPK3(), PhosphorylationThreonine, 201),
 Dephosphorylation(MAP2K2(), MAPK3(), PhosphorylationTyrosine, 203),
 Dephosphorylation(MAP2K2(), MAPK3(), PhosphorylationThreonine, 201),
 Dephosphorylation(MAP2K1(), MAPK1(), PhosphorylationTyrosine, 184),
 Dephosphorylation(MAP2K1(), MAPK1(), PhosphorylationThreonine, 182),
 Dephosphorylation(MAP2K2(), MAPK1(), PhosphorylationTyrosine, 184),
 Dephosphorylation(MAP2K2(), MAPK1(), PhosphorylationThreonine, 182),
```

Which appear to be artifactual. Strangely, they don't seem to turn up in the corresponding pathsbetween query among the same genes.

**Comment by bgyori** [comment]
I've been looking into this and I think this comes from a really weird bug in the Pathway Commons web service itself - the returned model is incorrect. 

After doing just `bp = ba.process_pc_neighborhood(['MAP2K1', 'MAP2K2', 'MAPK1', 'MAPK3'])`
I look at the model in bp.model which has this catalysis in it:
http://purl.org/pc2/7/Catalysis_d9e0c506bf5ca874d7d768475526fdca
according to bp.model, this catalysis controls
http://purl.org/pc2/7/BiochemicalReaction_e8e21806445399aac45c0d2e59065622

But that is incorrect! 

If you look up 
http://purl.org/pc2/7/Catalysis_d9e0c506bf5ca874d7d768475526fdca 
as a URL, you find that  it actually controls
http://purl.org/pc2/7/BiochemicalReaction_0b78fcfcf8ed3fb244cfdbbbb4ecfbb9

The dephosphorylation comes in because the wrong reaction that MEK as a controller is linked to happens to be one in which ERK is dephosphorylated.

**Comment by bgyori** [comment]
Fixed as of 5cee6d841e54d581fadfcbcb137207ae3572c26d by reimplementing Pathway Commons client to use the web service instead of cPath.
